framework/aster-frame/src/vm/options.rs

```diff
@@ -14,6 +14,7 @@ pub struct VmAllocOptions {
     nframes: usize,
     is_contiguous: bool,
     uninit: bool,
+    need_dealloc: bool,
```

Comment:
Maybe `need_dealloc` is more discouraged than we could imagine. I think it's better for a `cpu_local` global static to keep track of the allocated initial stack. They could have been recycled easier indeed in this form when the first process is created.